bootstrap.go

```diff
@@ -536,6 +541,14 @@ func upgradeToVer15(s Session) {
 	}
 }
 
+func upgradeToVer16(s Session) {
```

Comment:
We should not change the global `sql_mode` for existing data.
The user may have intentionally changed it to other values.

Reply:
At the beginning, I just want this be compatibility with mysql5.7 in default SQLMode. But I found this would cause be incompatibility with many cases. I think you are right.